chart/flux/values.yaml

```diff
@@ -22,13 +22,22 @@ helmOperator:
   tillerNamespace: kube-system
   logReleaseDiffs: false
   tls:
-    secretName: 'helm-client-certs'
+    secretName: "helm-client-certs"
     verify: false
     enable: false
-    keyFile: 'tls.key'
-    certFile: 'tls.crt'
-    caContent: ''
-
+    keyFile: "tls.key"
+    certFile: "tls.crt"
+    caContent: ""
+  # override Flux git settings
+  git:
+    url: ""
+    branch: ""
+    chartsPath: ""
+    # generate a SSH key named identity: ssh-keygen -q -N "" -f $HOME/.ssh/identity
```

Comment:
Sorry to quibble again .. this could overwrite another key that's in use. I would just advise people to create it wherever they are, since 1. that's likely to be within their home directory and 2. ssh-keygen will make the file mode 0400 anyway.
2.) counts for `/tmp` too, but I'd still be a bit wary of putting keys there.)

You could also recommend people delete the file after creating the secret, too. They can always create another key if the secret disappears.